src/js/components/landing/LandingContent.jsx

```diff
@@ -26,24 +26,30 @@ export default class LandingContent extends React.Component {
     constructor(props) {
         super(props);
 
+        this.didUnmount = false;
+
         this.state = {
             expanded: false,
+            type: this.props.type,
```

Comment:
Is there a particular reason we changed this to a state instead of a prop? It'll only change when the props change. If I'm missing the reason that's fine, but it seems like a lot of extraneous state setting.